python/tk_houdini/utils/parm_template_wrappers.py

```diff
@@ -21,27 +24,31 @@ class Parm(object):
     """
 
     @classmethod
-    def build_parm(cls, template):
+    def build_parm(cls, template, parent=None):
```

Comment:
IGNORABLE: Damn, I should have caught this much earlier but this method might be more explicitly named as
```python
def from_template(cls, template, parent=None):
```

Reply:
yeah, that is a good idea